packages/@tailwindcss-postcss/src/index.ts

```diff
@@ -48,114 +49,119 @@ function tailwindcss(opts: PluginOptions = {}): AcceptedPlugin {
     }
   })
 
+  let hasApply = false
+  let hasTailwind = false
```

Comment:
Would it be clearer if we initialized these without a value so it doesn't seem like these `false` values are actually important to how the code works?

`let hasApply, hasTailwind;`